### DISCUSSION TITLE
JavaScript indicator (like for muted tabs)

### DISCUSSION THREAD

**Discussion by pacien** [qutebrowser/qutebrowser#8023] [Q&A]
Is there a way to display whether JS is enabled or disabled for a given website
or tab?

For audio, I can put `{audio}` in `tabs.title.format` to display whether a tab
is muted or not.

I would like to have the same for JS, which I have disabled by default.

Also, is it possible to have indicators like these located elsewhere than in
the tab's title, such as the bottom status bar?

**Comment by toofar** [comment]
I don't think there in an out of the box way currently, [these](https://qutebrowser.org/doc/help/settings.html#tabs.title.format) are the options for the format strings in the title. There is this issue about adding a status bar indicator for various settings: https://github.com/qutebrowser/qutebrowser/issues/1052#issuecomment-246557721
That comment in particular has a workaround to change some hardcoded text in the title format, it doesn't take into account per-URL settings though.
For the status bar (apart from the PR linked to that issue) we have the text [widget type](https://qutebrowser.org/doc/help/settings.html#statusbar.widgets) these days which you could set in a similar way, again it would be difficult to do that taking into account per-URL settings though.
